pkg/operator/ceph/csi/spec.go

```diff
@@ -120,22 +128,38 @@ func StartCSIDrivers(namespace string, clientset kubernetes.Interface) error {
 		rbdProvisioner, cephfsProvisioner *apps.StatefulSet
 	)
 
+	// create an empty config map. config map will be filled with data
+	// later when clusters have mons
+	if CSIEnabled() {
```

Comment:
I think we can skip this check `StartCSIDrivers` is called only when csi is enabled. https://github.com/rook/rook/blob/master/pkg/operator/ceph/operator.go#L163

Reply:
sure